src/mail.sh

```diff
@@ -46,10 +46,39 @@ function mail_setup()
   return 0
 }
 
+function mail_verify()
+{
+  local count=0
+  local -a missing_conf
+  local -a min_conf=('user.name' 'user.email' 'sendemail.smtpencryption'
+    'sendemail.smtpserver' 'sendemail.smtpuser' 'sendemail.smtpserverport')
+  local set_confs
+
+  # say "${min_conf[*]}"
+  set_confs=$(git config --get-regexp 'user|sendemail\.smtp')
+  # say "*****$set_confs*****"
+  for config in "${min_conf[@]}"; do
+    if ! echo "$set_confs" | grep -cF "$config" - &> /dev/null; then
```

Comment:
Can we move this function outside this if? You can just check `"$?"`.
Also, add a comment explaining this line.